### PR TITLE
Rdm 13013

### PR DESCRIPTION
**Pull request by Thor-tech-of-metal**
RDM-13013: Include CaseAccessCategory in the standard search fields for the new ES Global Search Index
           [RDM-13013] (https://tools.hmcts.net/jira/browse/RDM-13013).

### CODE REVIEW

File: elastic-search-support/src/main/resources/globalSearchCasesMapping.json [hunk 1]
```diff
@@ -56,6 +56,18 @@
       "analyzer": "standard",
       "search_analyzer": "alphanumeric_string_analyzer"
     },
+    "case_access_category": {
```
Comment (danlysiak) [inline elastic-search-support/src/main/resources/globalSearchCasesMapping.json:59]:
```suggestion
    "CaseAccessCategory": {
```
A couple of points:
1. The field id should be `CaseAccessCategory`
1. It is a top level case data field so should be within the `data` object (alongside `SearchCriteria`, `caseManagementLocation` etc.)

File: elastic-search-support/src/main/resources/application.yml [hunk 2]
```diff
@@ -91,6 +91,7 @@ elasticsearch:
     created_date: ${elasticsearch.elasticMappings.defaultDate}
     last_modified: ${elasticsearch.elasticMappings.defaultDate}
     security_classification: ${elasticsearch.elasticMappings.defaultKeyword}
+    case_access_category:   ${elasticsearch.elasticMappings.defaultText}
```
Comment (danlysiak) [inline elastic-search-support/src/main/resources/application.yml:94]:
```suggestion
```
We should be able to delete this as these properties won't be getting used for the global search index

File: elastic-search-support/src/main/resources/globalSearchCasesMapping.json (hunk 1 above)
Comment (Thor-tech-of-metal) [reply elastic-search-support/src/main/resources/globalSearchCasesMapping.json]: DONE

File: elastic-search-support/src/main/resources/application.yml (hunk 2 above)
Comment (Thor-tech-of-metal) [reply elastic-search-support/src/main/resources/application.yml]: DONE

**Review by danlysiak — APPROVED** [review]
🎈

**Pull request merged**